frontend/requests/requestProcessingController.js

```diff
@@ -25,11 +25,19 @@
                 MessageService.showToast("Solicitação aceita!");
                 requestController.request.status = 'accepted';
                 requestController.hideDialog();
+                refreshUser();
```

Comment:
make tests to assert that the permissions are being added

Reply:
Done.